api-gateway/src/migrations/1702508933721-file-table.ts

```diff
@@ -1,30 +1,50 @@
-import { MigrationInterface, QueryRunner, TableColumn } from "typeorm";
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
 
-export class AddTimestampsToFile1702507595139 implements MigrationInterface {
+export class InitialFile1639427750672 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    const isTableExist = await queryRunner.hasTable('file');
 
-    public async up(queryRunner: QueryRunner): Promise<void> {
-        await queryRunner.addColumn(
-            'file', 
-            new TableColumn({
-                name: 'createdAt',
-                type: 'timestamp',
-                default: 'CURRENT_TIMESTAMP',
-            })
-        );
+    if (!isTableExist) {
+      await queryRunner.createTable(
+        new Table({
+          name: 'file',
+          columns: [
+            { name: 'id', type: 'int', isPrimary: true, isGenerated: true, generationStrategy: 'increment' },
+            { name: 'name', type: 'varchar' },
+            { name: 'pending_emails', type: 'int' },
+            { name: 'success_emails', type: 'int', default: 0 },
+            { name: 'failed_emails', type: 'int', default: 0 },
+            { name: 'userId', type: 'int' },
+            { name: 'status', type: 'varchar', default: 'loading' },
+            { name: 'endTime', type: 'timestamp', isNullable: true },
+            { name: 'emailText', type: 'text' },
+            { name: 'createdAt', type: 'timestamp', default: () => 'CURRENT_TIMESTAMP' },
+            { name: 'updatedAt', type: 'timestamp', default: () => 'CURRENT_TIMESTAMP' },
+          ],
+        }),
+        true
+      );
```

Comment:
You can use also this way:  await queryRunner.query(`CREATE TABLE IF NOT EXISTS \`File\` (\`id\` int NOT NULL AUTO_INCREMENT, \`name\` varchar(128) NOT NULL, \`useId\` int NOT NULL, PRIMARY KEY (\`id\`)) ENGINE=InnoDB`);